kuma/static/js/experiment-interactive-editor.js

```diff
@@ -0,0 +1,13 @@
+(function() {
+    'use strict';
+
+    var cop = new Mozilla.TrafficCop({
+        id: 'experiment-interactive-editor',
+        variations: {
+            'v=control': 50, // control
+            'v=test': 50 // interactive editor
+        }
+    });
+
+    cop.init();
+})();
```

Comment:
I think you'll need to:

1. Pass in ``window.Mozilla``
2. Set a longer cookie duration, if Kadir's metric is longer than a single visit.

https://github.com/mozilla/kuma/blob/c1d36a835fda3edb0b19f7348a77f681808a0d90/kuma/static/js/experiment-static-examples-on-top.js#L1-L14